### DISCUSSION TITLE
What libraries (if any) are required?

### DISCUSSION THREAD

**Discussion by Bug-Reaper** [brianhie/viral-mutation#15] [Q&A] [answered]
Such a awesome project! Was curious if any libraries/versions are needed to run the pipeline. I'd imagine some version of CUDA/CUDNN is required to enable GPU acceleration?

Cheers,

**Comment by brianhie** [comment, marked as answer]
Thanks @Bug-Reaper, some basic requirements are given in https://github.com/brianhie/viral-mutation/blob/master/requirements.txt. We do make use of CUDA-based GPU acceleration through tensorflow-gpu.
